Library/Formula/shc.rb

```diff
@@ -0,0 +1,23 @@
+class Shc < Formula
+  desc "Shell Script Compiler"
+  homepage "https://neurobin.github.io/shc"
+  url "https://github.com/neurobin/shc/archive/3.9.3a.tar.gz"
```

Comment:
Ah, I see. Sorry for not reading the whole PR.

It's cool if you want to add that back in, now that we have an upstream bug report to link to. Just include the URL to the bug report in a comment.